applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -126,6 +127,16 @@ struct le_audio_evt {
 typedef void (*le_audio_receive_cb)(const uint8_t *const data, size_t size, bool bad_frame,
 				    uint32_t sdu_ref, enum audio_channel channel_index);
 
+/**
+ * @brief Callback for getting the timestamp of the previously sent audio packet
+ *
+ * @note  Can be used for drift calculation/compensation
```

Comment:
And also for the "just in time" functionality. Should mention/explain this here

Reply:
That is up to the function that uses it, I believe. In theory the le_audio files doesn't need to know what the callbacks does with it